src/main/java/me/botsko/prism/listeners/PrismEntityEvents.java

```diff
@@ -296,25 +312,30 @@ public void onEntityDeath(final EntityDeathEvent event) {
     }
 
     /**
-     * @param event
+     * CreatureSpawnEvent.
+     * @param event CreatureSpawnEvent
      */
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void onCreatureSpawn(final CreatureSpawnEvent event) {
-        if (!Prism.getIgnore().event("entity-spawn", event.getEntity().getWorld()))
+        if (!Prism.getIgnore().event("entity-spawn", event.getEntity().getWorld())) {
             return;
+        }
         final String reason = event.getSpawnReason().name().toLowerCase().replace("_", " ");
-        if (reason.equals("natural"))
+        if (reason.equals("natural")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)